hypercert.json

```diff
@@ -0,0 +1,108 @@
+            "type": "string",
+            "description": "Title of the hypercert",
+            "maxLength": 256
+          },
+          "shortDescription": {
+            "type": "string",
+            "format": "uri",
+            "description": "(Optional) short blurb of the impact work done.",
+            "maxLength": 3000,
+            "maxGraphemes": 300
+          },
+          "description": {
+            "type": "string",
+            "format": "uri",
+            "description": "A description of the impact work done.",
+            "maxLength": 30000,
+            "maxGraphemes": 3000
+          },
+          "image": {
+            "type": "blob",
+            "accept": ["image/*"],
+            "maxSize": 5000000,
+            "description": "Blob reference to hypercert image/visual representation. Maximum size is in bytes"
+          },
+          "workScope": {
+            "type": "string",
+            "description": "Scope of the work performed",
+            "maxLength": 5000,
+            "maxGraphemes": 1000
+          },
+          "workTimeframeFrom": {
+            "type": "string",
+            "format": "datetime",
+            "description": "When the work began"
+          },
+          "workTimeFrameTo": {
+            "type": "string",
+            "format": "datetime",
+            "description": "When the work ended"
+          },
+          "evidence": {
+            "type": "array",
+            "description": "Supporting evidence, documentation, or external data URIs",
+            "items": {
+              "type": "object",
+              "required": ["uri"],
+              "properties": {
+                "uri": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "Evidence resource URI"
+                },
+                "title": {
+                  "type": "string",
+                  "maxLength": 256,
+                  "description": "Optional title to describe the nature of the evidence"
+                },
+                "description": {
+                  "type": "string",
+                  "maxLength": 3000,
+                  "maxGraphemes": 300,
+                  "description": "Optional short description explaining what this evidence demonstrates or proves"
+                }
+              }
+            },
+            "maxLength": 100
+          },
+          "contributors": {
+            "type": "array",
+            "description": "Contributors to this hypercert (names, pseudonyms, or DIDs)",
+            "items": {
+              "type": "string",
+              "maxLength": 2560,
+              "maxGraphemes": 256
+            }
+          },
+          "rights": {
+            "type": "string",
+            "description": "Rights keyword (as defined in app.certified.hypercert.rights)",
+            "maxLength": 10
+          },
+          "location": {
+            "type": "string",
+            "format": "at-uri",
```

Comment:
at:// URIs don't include the CID, so this would allow the content of that location record to change.  I think the way to do this is probably:

```
  "type": "ref",
  "ref": "com.atproto.repo.strongRef"
```

You would think there'd be a way to enforce the link to be to a particular lexicon type, but apparently this isn't supported:

https://github.com/bluesky-social/atproto/discussions/3661